CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v0.7.2
+
+* Bug fix: reverify using fissix instead of ast to preserve the ability to modify code
+  that is incompatible with the host version of python (#77)
```

Comment:
I'm sure you didn't necessarily know the PR number at the time you wrote this, but can you either update it or add #78 so that it points to the PR that made the change?